src/shader_param.rs

```diff
@@ -185,7 +185,7 @@ fn node_to_var_type(cx: &mut ext::base::ExtCtxt,
         Ok(Param::Texture) => "VarTexture",
         Ok(Param::Special) => return quote_ty!(cx, Option<()>),
         Err(ParamError::DeprecatedTexture) => {
-            cx.span_err(span, "Use gfx::shade::TextureParam for texture vars instead of gfx::shade::TextureHandle");
+            cx.span_err(span, "Use gfx::render::shade::TextureParam for texture vars instead of gfx::Texture");
```

Comment:
no need to have `gfx::render::shade` here, please leave it as `gfx::shade::TextureParam`